src/daemon/configfile.c

```diff
@@ -628,8 +629,11 @@ static int cf_include_all (oconfig_item_t *root, int depth)
 			return (-1);
 
 		/* Now replace the i'th child in `root' with `new'. */
-		if (cf_ci_replace_child (root, new, i) < 0)
+		if (cf_ci_replace_child (root, new, i) < 0) {
+			sfree (new->values);
```

Comment:
How did you find this?  Should we make it a separate pull request/commit for collectd?  Thoughts?